flashinfer/comm/allreduce.py

```
@@ -664,22 +668,41 @@ def _flatten_checked(t, name):
+                        output=norm_out,
+                        residual_out=residual_out,
+                        launch_with_pdl=launch_with_pdl,
+                        output_scale=scale_factor,
+                        layout_code=layout_code,
+                        quant_type=pattern_traits.quant_type,
+                    )
+                )
+                return quant_result
+            else:
+                # Call the MNNVL fusion function
+                norm_result, residual_result = trtllm_mnnvl_fused_allreduce_add_rmsnorm(
```

Ruff (0.14.10)

Unpacked variable `residual_result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)